src/Components/Admins/Form/index.js

```diff
@@ -0,0 +1,109 @@
+import { useState, useEffect } from 'react';
+import styles from './form.module.css';
+
+const params = new URLSearchParams(window.location.search);
+const adminId = params.get('_id');
+let fetchMethod = 'POST';
+let url = `${process.env.REACT_APP_API}/admins`;
+
+function Form() {
+  const [nameValue, setNameValue] = useState('');
+  const [usernameValue, setUsernameValue] = useState('');
+  const [passwordValue, setPasswordValue] = useState('');
+
+  const getAdminById = () => {
+    fetch(`${process.env.REACT_APP_API}/admins?_id=${adminId}`)
+      .then((response) => {
+        if (response.status !== 200 && response.status !== 201) {
+          return response.json().then(({ message }) => {
+            throw new Error(message);
+          });
+        }
+        return response.json();
+      })
+      .then((response) => {
+        const { name, username, password } = response.data[0];
+        setNameValue(name);
+        setUsernameValue(username);
+        setPasswordValue(password);
+      })
+      .catch((error) => {
+        console.log('err', error);
+      });
+  };
+
+  useEffect(() => {
+    if (adminId) {
+      url += `/${adminId}`;
+      fetchMethod = 'PUT';
+      getAdminById();
+    }
+  }, []);
+
+  const save = (e) => {
+    e.preventDefault();
+    fetch(url, {
+      method: fetchMethod,
+      headers: {
+        'Content-Type': 'application/json'
+      },
+      body: JSON.stringify({
+        name: nameValue,
+        username: usernameValue,
+        password: passwordValue
+      })
+    })
+      .then((response) => {
+        if (response.status !== 200 && response.status !== 201) {
+          return response.json().then(({ message }) => {
+            throw new Error(message);
+          });
+        }
+        window.location.href = '/admins';
+      })
+      .catch((error) => {
+        console.log('err', error);
```

Comment:
remove the console.log and show the error in the UI

Reply:
all console.log() at admins were deleted